README.md

```diff
@@ -165,6 +165,18 @@ Here are the additional props that `ImmutableListView` accepts:
 | `sectionHeaderHasChanged` | `func` | `(prevSectionData, nextSectionData) => false` | Only needed if your section header is dependent on your row data (uncommon; see [`ListViewDataSource`'s constructor](https://facebook.github.io/react-native/docs/listviewdatasource.html#constructor) for details). |
 | `renderEmpty` | `func` | `(originalProps) => `[`React.PropTypes.element`](https://facebook.github.io/react/docs/typechecking-with-proptypes.html#react.proptypes) | If your data is empty (e.g. `null`, `[]`, `{}`) and this prop is defined, then the result of this function will be rendered instead. For example, you can use the [`EmptyListView` component](#emptylistview) for this. |
 
+## Functions
+
+The references of ListView and VirtualizedList are available with two functions:
+
+- `getListView` and `getVirtualizedList`
+
+Theses references will allow you to access to all functions of the mother component like:
+
+| ListView | VirtualizedList |
+|-----------|-----------|
+| `setNativeProps`, `getInnerViewNode`, `getScrollResponder`, `getScrollableNode` | `getScrollResponder`, `getScrollableNode`|
```

Comment:
I'd prefer not to document these since they're really not meant for public access unless necessary and used with caution. Let's just say "These references allow you to access any other methods on the List component that you might need."